docs/architecture/TypeScript-ESTree.mdx

```diff
@@ -31,6 +31,12 @@ Parses the given string of code with the options provided and returns an ESTree-
 
 ```ts
 interface ParseOptions {
+  /**
+   * Prevents the parser from throwing an error if it receives an invalid AST from TypeScript.
+   * This case only usually occurs when attempting to lint invalid code.
+   */
+  allowInvalidAST?: boolean;
```

Comment:
@bradzacher just raising this, I last-minute renamed the property from `errorOnInvalidAST` to `allowInvalidAST` to reflect the change in error throwing from off-by-default to on-by-default. Do you have feedback on that change and/or the new name?